vector/src/main/res/layout/activity_medias_previewer.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/riot_primary_background_color_black">
+
+    <ImageView
+        android:id="@+id/image_previewer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_centerInParent="true"
+        android:scaleType="fitCenter"
+        android:visibility="gone"/>
```

Comment:
Please add a line: `tools:visibility="visible"` to help for the maintenance (and to the other gone Views below)